specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2019-05-01/machineLearningServices.json

```diff
@@ -1248,6 +1343,156 @@
         }
       }
     },
+    "QuotaBaseProperties": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Specifies the resource ID."
+        },
+        "type": {
+          "type": "string",
+          "description": "Specifies the resource type."
+        },
+        "quota": {
+          "type": "integer",
+          "format": "int32",
+          "title": "Quota.",
+          "description": "The workspace level quota."
+        }
+      },
+      "description": "The properties for Quota update or retrieval."
+    },
+    "QuotaUpdateParameters": {
+      "properties": {
+        "value": {
+          "description": "The list for update quota.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/QuotaBaseProperties"
+          }
+        }
+      },
+      "description": "Quota update parameters."
+    },
+    "UpdateWorkspaceQuotasResult": {
+      "properties": {
+        "value": {
+          "description": "The list of workspace quota update result.",
+          "items": {
+            "$ref": "#/definitions/UpdateWorkspaceQuotas"
+          },
+          "readOnly": true,
+          "type": "array"
+        },
+        "nextLink": {
+          "description": "The URI to fetch the next page of workspace quota update result. Call ListNext() with this to fetch the next page of Workspace Quota update result.",
+          "readOnly": true,
+          "type": "string"
+        }
+      },
+      "description": "The result of update workspace quota."
+    },
+    "UpdateWorkspaceQuotas": {
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Specifies the resource ID."
+        },
+        "type": {
```

Comment:
should this be an enum? How many types are supported?

Reply:
for "id" and "type", we will follow the same as usages contract: https://microsoft-my.sharepoint.com/:w:/p/gauravbh/EYPNGVUmC3hGskyHRaTk4cABlwkm6ziCMnmqLMj0QSJ0Vg?rtime=BW9QFs311kg